author: Stefan Wolfsheimer, SURF
disable nc_set_log_level as it is not supported in netCDF/4.9.3-gompi-2025a


diff -ruN netcdf-cxx4-4.3.1.orig/examples/pres_temp_4D_plugin_wr.cpp netcdf-cxx4-4.3.1/examples/pres_temp_4D_plugin_wr.cpp
--- netcdf-cxx4-4.3.1.orig/examples/pres_temp_4D_plugin_wr.cpp	2019-09-12 20:34:58.000000000 +0200
+++ netcdf-cxx4-4.3.1/examples/pres_temp_4D_plugin_wr.cpp	2025-08-06 13:25:16.438529000 +0200
@@ -71,7 +71,7 @@
 
 int main()
 {
-  nc_set_log_level(5);
+  // nc_set_log_level(5);
   // We will write latitude and longitude fields.
    float lats[NLAT],lons[NLON];
 
@@ -103,7 +103,7 @@
 
       // Create the file.
       NcFile test(FILE_NAME, NcFile::replace);
-      nc_set_log_level(5);
+      // nc_set_log_level(5);
       // Define the dimensions. NetCDF will hand back an ncDim object for
       // each.
       NcDim lvlDim = test.addDim(LVL_NAME, NLVL);
